### PR TITLE
add volto translation jobs for master and 16.x.x

### PR DESCRIPTION
**Pull request by erral**
Now that we have Volto 16.x.x LTS in its own branch, it is worth to have the translation status for it and `master` also.

Can you review if the syntax is OK and if so deploy it?

### CODE REVIEW

File: jobs/jobs.yml [hunk 1]
```diff
@@ -178,9 +178,19 @@
...
-        - 'volto-translations'
+        - 'volto-translations-{branch}'
+
+- project:
+    name: Translations of Volto (master)
+    branch:
+        - 'master'
+    jobs:
+        - 'volto-translations-{branch}'
+
```
Comment (gforcada) [inline jobs/jobs.yml:193]: You don't need to create 2 `project:` keys, something like this is enough:

```yml
project:
    name: Translations Volto
    branch:
        - '16.x.x'
        - 'master'
    jobs:
        - 'volto-translations-{branch}'
```
🍀

**Review by gforcada — CHANGES_REQUESTED** [review]
Great job, one minimal fix and we can deploy 🚀

**Comment by erral** [comment]
done!

**Review by gforcada — APPROVED** [review]

**Comment by gforcada** [comment]
@erral changes are tested, jobs have been pushed, and are [already green](https://jenkins.plone.org/view/Translations/) 🍀 

I disabled the _old job_ ([this one](https://jenkins.plone.org/view/Translations/job/Volto%20translations/)), let me know if it can be 💥

**Comment by erral** [comment]
yes that's right, you can disable the old one. thanks !

**Pull request merged**